xbmc/interfaces/legacy/File.h

```diff
@@ -38,6 +38,7 @@ namespace XBMCAddon
     ///
     ///
     ///--------------------------------------------------------------------------
+    /// @python_v19 Added context manager support
```

Comment:
maybe update the example as well?

Reply:
@enen92 I added new examples, so one still has the examples for older Kodi versions.